### PR TITLE
*: remove member verb

### PR DESCRIPTION
**Pull request by s-urbaniak**
## Summary

It was being removed from virtual workspace handling hence unused.

## Related issue(s)

https://github.com/kcp-dev/kcp/pull/1685

### CODE REVIEW

**Review by davidfestal — APPROVED** [review]

File: contrib/demos-unmaintained/demo/prototype2-script/user-rbac.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ rules:
 - apiGroups: ["tenancy.kcp.dev"]
   resources: ["clusterworkspaces/content"]
   resourceNames: ["demo"]
-  verbs: ["access","member"]
```
Comment (stevekuznetsov) [inline contrib/demos-unmaintained/demo/prototype2-script/user-rbac.yaml:10]: next time we should acknowledge the smell of calling a noun, a verb ;)

Comment (s-urbaniak) [reply contrib/demos-unmaintained/demo/prototype2-script/user-rbac.yaml:10]: lol, yes :-) in fact, i do believe i found precendences in k8s using nouns as well :-|

**Pull request merged**